docs/UserGuide.adoc

```diff
@@ -46,100 +47,210 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
+** Valid input:
+
+* d/**DOCTOR** - doctor in charge of a patient
+** Applicable to: patients
+
+
+* exp/**EXPIRY** - expiry date of a donor's organ
+** Applicable to: donors
+
+* ext/**EXTENSION NUMBER** - extension number of a doctor
+** Applicable to: doctors
+** Valid input:
 
 === Viewing help : `help`
 
 Format: `help`
 
 === Adding a person: `add`
 
-Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Adds a patient, doctor, or donor to OTM with the attributes given, depending on what is specified.
+
+Format for adding patients: `add patient n/**NAME** g/**GENDER** b/**BLOODTYPE** p/**PRIORITY** c/**CONTACT** +
+id/**NRIC** d/**DOCTOR** a/**AGE**`
+
+Format for adding donors: `add donor n/**NAME** g/**GENDER** b/**BLOODTYPE** c/**CONTACT** +
+id/**NRIC** exp/**EXPIRY** a/**AGE**`
+
+Format for adding doctors: `add doctor n/**NAME** ext/**EXTENSION NUMBER** c/**CONTACT** +
+id/**NRIC**`
 
 [TIP]
-A person can have any number of tags (including 0)
+View <<List of attributes>> to find out what each attribute mean.
 
 Examples:
 
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add patient n/John Doe g/M p/high b/A a/29 id/S9988330G d/1234 c/98765432`
+* `add donor n/Betsy g/F c/99998888 b/AB p/1234567 exp/23/11/2020 a/21 id/S2345678H`
+* `add doctor ext/1234 c/91054444 id/S5678908D`
+
+**Form Mode**
+
+Besides typing the command out in full to add a person, OTM allows for adding a person by filling in a form.
 
-=== Listing all persons : `list`
+Format: `add [patient/doctor/donor]`
 
-Shows a list of all persons in the address book. +
-Format: `list`
+OTM will generate a form for the user to input the person's attributes.
+
+**Detecting duplicates**
+
+When adding a person, OTM will detect if the input is a duplicate of another person in the list.
+A duplicate is defined as: Two persons of the same type who have the same NRIC.
+
+When a duplicate is detected, OTM will indicate that there is such a person in OTM so the inputs
+should be changed.
+
+=== Listing persons : `list`
+
+Shows a list of all doctors, donors, or patients in OTM, depending on the type of person specified.
+Format: `list [patient/doctor/donor]`
+
+Examples:
+
+* `list patient`
+* `list doctor`
+* `list donor`
 
 === Editing a person : `edit`
 
-Edits an existing person in the address book. +
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+Edits an existing person's information. +
+Format: `edit [patient/doctor/donor] **INDEX** [attributes]`
 
 ****
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index *must be a positive integer* 1, 2, 3, ...
+* Edits the person at the specified `**INDEX**`. The index refers to the index number shown in the list of the +
```

Comment:
instead of index, should change to nric?

Reply:
Modified at my latest commit.